automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -282,7 +300,7 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
 
       // Verify notebook error
       val caught = the [RestException] thrownBy {
-        Leonardo.notebooks.getTree(googleProject, clusterName)
+        Notebook.getApi(googleProject, clusterName)
```

Comment:
Could you explain this change?

Wondering if this is what is causing the tests to fail.